src/views/rebenchdb.R

```diff
@@ -139,16 +139,29 @@ get_profile_availability <- function(rebenchdb, hash_1, hash_2) {
   }
 
   convert_double_array <- function(x) {
-      vapply(convert_array(x),as.double,double(1))
+      sapply(convert_array(x),as.double)
```

Comment:
vapply will not take a value greater than length 1. so i had to switch to lapply